src/renderer/redux/selectors/media.js

```diff
@@ -0,0 +1,15 @@
+import * as settings from "constants/settings";
+import { createSelector } from "reselect";
+
+const _selectState = state => state.media || {};
+
+export const selectMediaPaused = createSelector(
+  _selectState,
+  state => state.paused
+);
+
+export const makeSelectMediaPositionForUri = uri =>
+  createSelector(_selectState, state => {
+    const id = uri.split("#")[1];
```

Comment:
will this work when the uri does not contain a `#` such as in the case of `lbry://one`?